content/docs/latest/installing/community-platforms/scaleway.md

```diff
@@ -77,7 +77,7 @@ The `coreos-metadata.service` saves metadata variables to `/run/metadata/flatcar
 Boot the machine with the CLI, referencing the snapshot ID from the import step above and your [Ignition file from Butane][butane-configs]:
 
 ```shell
-$ INSTANCE_ID=$(scw instance server create image=none root-volume=l:"${SNAPSHOT_ID}" cloud-init=@./config.json --output=json | jq -r .id)
+$ INSTANCE_ID=$(scw instance server create image=none root-volume=l:"${SNAPSHOT_ID}" cloud-init=@./config.json type=DEV1-S --output=json | jq -r .id)
```

Comment:
Should the meaning of DEV1-S be explained or is it something clear for scaleway users?

Reply:
Well, I guess that if someone is trying to use scaleway then the person should know about the machine types.